cluster/gce/gci/configure-helper.sh

```diff
@@ -577,6 +577,31 @@ rules:
       - group: "" # core
         resources: ["events"]
 
+  # node and pod status calls from nodes are high-volume and can be large, don't log responses for expected updates from nodes
+  - level: Request
```

Comment:
if required to hit scale targets, we could even go to metadata-level logging on these, though I'd prefer to see the request content